!GameTexts/NewTexts.xml

```diff
@@ -409,24 +401,24 @@
 			<Text></Text>
 		</Row>
 		<Row Tag="TXT_KEY_RESOURCE_EGYPT_FLAX_TEXT">
-			<Text>Flax is a food and fiber crop best known for its use in the production of linen. The crop was first domesticated in the Fertile Crescent and was cultivated extensively in Egypt, where the fabric became tightly bound to the material culture. Linen has a smooth texture, high lustre, and is cool to the touch. These characteristics linked it to religious and ceremonial purity within Egyptian rituals. Priests wore linen fabrics exclusively, and mummies were wrapped in linen in preparation for the afterlife. Today, more durable textiles like cotton have largely replaced linen as a fabric, but the flax seed is still widely used for food and cooking oil.</Text>
+			<Text>Flax is a food and fiber crop best known for its use in the production of linen. The crop was first domesticated in the Fertile Crescent and was cultivated extensively in Egypt, where the fabric became tightly bound to the local culture. Linen has a smooth texture, high lustre, and is cool to the touch. These characteristics linked it to religious and ceremonial purity within Egyptian rituals. Priests wore linen fabrics exclusively, and mummies were wrapped in linen in preparation for the afterlife. Today, more durable textiles like cotton have largely replaced linen as a fabric, but the flax seed is still widely used for food and cooking oil.</Text>
 		</Row>
 		
 		<!-- Grande Ecole -->
 		<!-- Created by adan_eslavo 10/11/2017 -->
 		<Row Tag="TXT_KEY_BUILDING_FRANCE_GRANDE_ECOLE">
-			<Text>Grande École</Text>
+			<Text>Grande Ecole</Text>
```

Comment:
French "E" is accepted by game. Look at Egyptian city names. We can add it if you want.

Reply:
If it can be read by Civ then it should stay, but it will have to be consistent. Need to check civilopedia, help and strategy files too